src/GovUk.Education.ExploreEducationStatistics.Data.Importer/Services/ImporterMetaService.cs

```diff
@@ -76,7 +77,8 @@ public static MetaRow GetMetaRow(List<string> cols, DataRow row)
                 FilterGroupingColumn = values[3],
                 FilterHint = values[4],
                 IndicatorGrouping = values[5],
-                IndicatorUnit = EnumUtil.GetFromString<Unit>(values[6] ?? "")
+                IndicatorUnit = EnumUtil.GetFromString<Unit>(values[6] ?? ""),
+                DecimalPlaces = values[7] == null ? (int?) null : int.Parse(values[7])
```

Comment:
Do we actually do any meta file validation to ensure it's in the correct format? I'm guessing we don't report any problems back to the user if there is an issue?

Reply:
If it's non-numeric then it'll be caught by the upload validation but no val on max or min values.